src/NServiceBus.Transport.RabbitMQ.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -73,4 +73,8 @@ namespace NServiceBus.Transport.RabbitMQ
         System.Threading.Tasks.ValueTask SetupSubscription(RabbitMQ.Client.IChannel channel, NServiceBus.Unicast.Messages.MessageMetadata type, string subscriberName, System.Threading.CancellationToken cancellationToken = default);
         System.Threading.Tasks.ValueTask TeardownSubscription(RabbitMQ.Client.IChannel channel, NServiceBus.Unicast.Messages.MessageMetadata type, string subscriberName, System.Threading.CancellationToken cancellationToken = default);
     }
+    public static class NativeIntegrationDispatchPropertiesExtensions
+    {
+        public static void SetContentType(this NServiceBus.Transport.DispatchProperties dispatchProperties, string contentType) { }
```

Comment:
By looking at the dispatch properties it means whenever a transport operation is available this method can be set. For example, it is possible to interact with PendingTransportOperations over the extension bag. If you do that the content type can be set which sets the property in the dispatch properties. Those get serialized into the outbox message which means the information is persisted into the outbox. Is that desired? If yes should this be documented at least with a comment somewhere?